index.js

```diff
@@ -255,6 +255,15 @@ class Bot extends EventEmitter {
     }
 
     /**
+     * Get the channel history
+     * @param {string} channel
```

Comment:
It's the name of the channel as it is specified on the docs: [channels.history](https://api.slack.com/methods/channels.history)